src/clientsettingsdlg.cpp

```diff
@@ -196,12 +199,12 @@ CClientSettingsDlg::CClientSettingsDlg ( CClient* pNCliP, CClientSettings* pNSet
                                 tr ( "Some sound card drivers do not allow the buffer delay to be changed "
                                      "from within the application. "
                                      "In this case the buffer delay setting is disabled and has to be "
-                                     "changed using the sound card driver. On Windows, press the "
+                                     "changed using the sound card driver. On Windows, use the "
                                      "ASIO Device Settings button to open the driver settings panel. On Linux, "
-                                     "use the Jack configuration tool to change the buffer size." ) +
+                                     "use the JACK configuration tool to change the buffer size." ) +
                                 "<br>" +
-                                tr ( "If no buffer size is selected and all settings are disabled, an "
-                                     "unsupported buffer size is used by the driver. The application "
+                                tr ( "If no buffer size is selected and all settings are disabled, this means an "
+                                     "unsupported buffer size is in use by driver. The application "
                                      "will still work with this setting but with restricted "
                                      "performance." ) +
```

Comment:
Should we say "Jamulus will still work with this setting, but may have restricted performance." ?

Reply:
Ok. See next commit